docs/examples/pytorch/resnet50/main.py

```diff
@@ -282,7 +285,8 @@ def main():
             }, is_best)
             if epoch == args.epochs - 1:
                 print('##Top-1 {0}\n'
-                      '##Top-5 {1}'.format(prec1, prec5))
+                      '##Top-5 {1}\n'
+                      '##Perf  {2}'.format(prec1, prec5, args.world_size * args.batch_size / total_time.avg))
```

Comment:
knitpick: `args.world_size * args.batch_size` could be extracted to separate constant